### ISSUE TITLE
Spatial Support when using .NET 5

### ISSUE THREAD

**Issue by henrikdahl8240** [OData/AspNetCoreOData#197]
Referring to https://devblogs.microsoft.com/odata/how-to-consume-sql-spatial-data-with-web-api-v2-2-for-odata-v4/ it seems like this approach does not work with .NET 5 as Microsoft.Spatial presumably does not work with .NET 5.

What should be done in order to use spatial classes like Edm.GeometryPoint when using .NET 5 or spatial is not supported anymore?

**Comment by xuzhg** [comment]
@henrikdahl8240 Thanks for reporting this. However, would you please share more about "Microsoft.Spatial presumably does not work with .NET 5"?

**Comment by henrikdahl8240** [comment]
@xuzhg Unfortunately I have realized that what I wrote was wrong. I looked at it very late in the evening and I was sleepy, so I think I did some mistake.. It looked to me, that it was not possible to reference the assembly and effectively make it visible, but I was wrong, I am sorry. In fact I realized it early next day.
Is it possible to remove this whole thread, so it will not disturb?
I am sorry I was wrong on this issue.
